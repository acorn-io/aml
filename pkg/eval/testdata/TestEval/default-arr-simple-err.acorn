define empty: {
    arr: ["foo", "bar"]
}

custom: empty({
    arr: ["blah", 5]
})