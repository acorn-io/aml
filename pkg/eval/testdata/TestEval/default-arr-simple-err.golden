"schema violation key arr: expected kind string but got kind number [path arr] [schema path empty]: default-arr-simple-err.acorn:5:14 (2:10<-2:5<-5:14)"
